### ISSUE TITLE
Refresh option in Winmerge

### ISSUE THREAD

**Issue by ganer9** [WinMerge/winmerge#2268]
Refresh Selected or Ctrl + F5 refreshes all folders in the original compare.

How do I Refresh Selected

Thank You

**Comment by sdottaka** [comment]
When you click Refresh Selected menu item, the entire window is redrawn and the selection position returns to the beginning, but only the selected items should be re-compared. Are you complaining about the selection moving back to the beginning, or are all items actually being recompared?

**Comment by ganer9** [comment]
I am complaining all items are recompared SOMETIMES

**Comment by sdottaka** [comment]
I am currently unable to reproduce that all items are recompared. If you know the conditions to reproduce it, please let me know.

**Comment by ganer9** [comment]
Hello

I m seeking a software advice.
How do I reach you?

Thank you indeed

     On Saturday 23 March, 2024 at 05:34:10 pm IST, Takashi Sawanaka ***@***.***> wrote:  
 
 


When you click Refresh Selected menu item, the entire window is redrawn and the selection position returns to the beginning, but only the selected items should be re-compared. Are you complaining about the selection moving back to the beginning, or are all items actually being recompared?

**Comment by sdottaka** [comment]
Of course, please contact us at this github URL https://github.com/WinMerge/winmerge/issues/2338.

By the way, isn't this problem related to the following issue?

https://github.com/WinMerge/winmerge/issues/2338

That is, the problem is that when comparing selected items, incorrect file names are displayed in the progress bar.

This has been fixed in the latest version, so could you please check the latest version?

**Comment by sdottaka** [comment]
> Of course, please contact us at this github URL https://github.com/WinMerge/winmerge/issues/#2338.

The URL was incorrect. The correct URL is as follows:

https://github.com/WinMerge/winmerge/issues/2268

**Comment by ganer9** [comment]
The question I've is unrelated to Winmerge

I will check in the latest version

**Comment by sdottaka** [comment]
Sorry. I completely misread it.

You can contact me at the email address on the Japanese Takashi Sawanaka line at https://winmerge.org/translations/?lang=en or at the email address at https://github.com/sdottaka, but I don't think I can give you any good advice on anything other than WinMerge.
